ironfish/src/sdk.ts

```diff
@@ -204,7 +204,11 @@ export class IronfishSdk {
     }
 
     if (this.config.get('enableRpcTcp')) {
-      const protectedNamespaces = [ApiNamespace.account, ApiNamespace.config]
+      const protectedNamespaces = [
+        ApiNamespace.account,
+        ApiNamespace.config,
+        ApiNamespace.transaction,
+      ]
```

Comment:
This seems good, we'll have to put out an announcement to the community regarding this they are likely to no longer be able to send tranactions.

Reply:
If i want to use `transaction/sendTransaction` over tcp, i would first use `accounts/getBalance`. 
Since `accounts` routes are protected before, there is no necessary to put out an announcement for `sendTransaction`.

Reply:
Good point.